model-zoo-api/dict/generate_model_versions.py

```diff
@@ -0,0 +1,42 @@
+from pathlib import Path
+import re
+from collections import defaultdict
+
+
+# identify list of onnx models in model Zoo
+model_list = []
+
+def get_a_list():
```

Comment:
I don't think this function is necessary... why use defaultdict?

Reply:
Yea, I can change it to `dict = defaultdict(list)` instead, and delete the `get_a_list()` function. The reason why we used defaultdict was because we need defaultdict to initialize the first value type, or else it would be empty.